README.md

```diff
@@ -32,13 +32,30 @@ To install the New Relic Helm charts, add the official repository first:
 helm repo add newrelic https://helm-charts.newrelic.com
 ```
 
-You can list all the available charts from the `newrelic` repository using [`helm search`][helm-search]:
+To install the bundle, create a values file that looks like this: 
+```yaml
+global:
+  licenseKey: YOUR_LICENSE_KEY
+  cluster: YOUR_CLUSTER_NAME
+kubeEvents:
+  enabled: true
+webhook:
+  enabled: true
+prometheus:
+  enabled: true
+logging:
+  enabled: true
+ksm:
+  enabled: true
+```
 
-```sh
-helm search repo newrelic/
+Then, run `helm updagre`:
```

Comment:
```suggestion
Then, run `helm upgrade`:
```

Reply:
Solved in [#4f2332e](https://github.com/newrelic/helm-charts/pull/795/commits/4f2332e6fc57c5e9842ddb8e958681644a537814)